extensions/adobe/experience/decisioning/offerItem.schema.json

```diff
@@ -0,0 +1,123 @@
+{
+  "meta:license": [
```

Comment:
For the properties in this class, please use the fully qualified namespace as in the field group above
https://ns.adobe.com/experience/decisioning/......

For an internal Adobe solution we only use "xdm" namespace for properties defined in a data type.